doc/tutorials/snippet-sharing-rest-service.md

```diff
@@ -121,7 +121,7 @@ After that you need to set additional configuration parameters that have no defa
 It is recommended to set them for the system namespace if you will use a tool like
 `systemctl` to manage the services. (For the `api/description` keys, see below!)
 ```
-> kdb set system/sw/elektra/restbackend/#0/current/backend/jwt/encryption_key "use_a_secret_key_here!"
+> kdb set system/sw/elektra/restbackend/#0/current/backend/jwt/encryptionkey "use_a_secret_key_here!"
```

Comment:
Maybe  `encryption/secret`? @petermax2 uses masterpassword, which you should also consider for consistency.